tests/e2e/specs/onboarding/step-3-complete-campaign.test.js

```diff
@@ -409,14 +409,14 @@ test.describe( 'Complete your campaign', () => {
 
 			test.describe( 'Budget recommendations', () => {
 				test.beforeEach( async () => {
-					await page.evaluate( () => window.sessionStorage.clear() );
 					await setupAdsAccountPage.mockAdsAccountIncomplete();
 					await setupBudgetPage.fulfillBillingStatusRequest( {
 						status: 'approved',
 					} );
 					await completeCampaign.mockCompleteAdsSetup();
 					await completeCampaign.fulfillBudgetRecommendations();
 					await completeCampaign.goto();
+					await page.evaluate( () => window.sessionStorage.clear() );
```

Comment:
These two `page.evaluate()` calls (see below) were failing due to a race condition with page navigation events when I was testing locally. Any concerns with moving this until after we're on the page state we expect?

Reply:
I am unable to reproduce the issue you encountered, so I cannot verify whether moving this line is the appropriate solution. It seems counterintuitive here, but I don't think it's a big problem if you confirm it's necessary.

Reply:
I'm unsure why the order matters in this case, but waiting until after the tests have navigated to the correct screen before clearing sessionStorage seems to make this more consistent.